Dockerfile

```diff
@@ -1,3 +1,5 @@
+# Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
```

Comment:
Added copyright notices throughout. Sorry, hijacking my own PR. But they need to be in there.